diff --git a/necsim-rust/Cargo.toml b/necsim-rust/Cargo.toml
--- a/necsim-rust/Cargo.toml
+++ b/necsim-rust/Cargo.toml
@@ -36,6 +36,13 @@ members = [
     "rust-cuda",
     "rust-cuda/rust-cuda-derive",
 
+    "analysis/rng/randomness",
+    "analysis/rng/hash",
+    "analysis/rng/correlation",
+    "analysis/performance/exponential",
+    "analysis/performance/exponential/kernel",
+    "analysis/performance/reporting",
+
     "third-party/array2d-no-std",
     "third-party/contracts",
     "third-party/float-next-after-no-std",
