diff --git a/necsim-rust/Cargo.toml b/necsim-rust/Cargo.toml
--- a/necsim-rust/Cargo.toml
+++ b/necsim-rust/Cargo.toml
@@ -31,6 +31,13 @@ members = [
     "rustcoalescence/algorithms/cuda",
     "rustcoalescence/algorithms/cuda/gpu-kernel",
     "rustcoalescence/algorithms/cuda/cpu-kernel",
+
+    "analysis/rng/randomness",
+    "analysis/rng/hash",
+    "analysis/rng/correlation",
+    "analysis/performance/exponential",
+    "analysis/performance/exponential/kernel",
+    "analysis/performance/reporting",
 ]
 
 default-members = [
